cmd/ci-op-configs-mirror/main.go

```diff
@@ -87,6 +88,10 @@ func main() {
 			privateBaseImages(rbc.BaseImages)
 		}
 
+		if len(rbc.BaseRPMImages) > 0 {
+			privateBaseImages(rbc.BaseRPMImages)
```

Comment:
nit: functions with side-effects are hard to follow and we should prefer not to use them, instead using functions with clear data flow